src/SearchExamples.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const SearchExamples = ({links}) =>
+  <p style={{fontSize: `smaller`}}>Examples:
+    {
+      links.map(
+        (link, index) =>
+          <span key={index}>
+            {index === 0 && ` `}<a href={link.url}>{link.text}</a>{index < links.length - 1 && `, `}
```

Comment:
The last link should not be followed by a comma I believe.

Reply:
It won’t be, notice that the right hand side is `links.length -1` instead of `links.length`.

Reply:
The comma is shown on the SC homepage actually.

![screen shot 2018-11-23 at 11 52 38](https://user-images.githubusercontent.com/33519183/48942191-4faf2f00-ef16-11e8-96ca-d3b845997d4a.png)

Reply:
It's weird since I try the same test `searchExamples` data in the local search form branch, which works fine. However, it appears differently on the atlas branch.

Reply:
Oh, yes, that’s because the bundles were from the penultimate release. If you generate the bundles yourself and try it out it’ll be ok.